### ISSUE TITLE
How to add meta information for page

### ISSUE THREAD

**Issue by dinesh2017** [jadjoubran/laravel5-angular-material-starter#484]
* **I'm submitting a ...**
- [ ] bug report
- [ ] feature request
- [x] support request

* **Provide information & explanation**
Hi, Right now i'm facing problem with assign meta information. I'm using like **ngMet**a module. It was added information successfully. **but not showing while facebook share option**. Showing only default title & meta information. please help me in this issue 

Thank you,

* **Laravel & Angular version:** 3.2.0

**Comment by dinesh2017** [comment]
any one please help me....

**Comment by jadjoubran** [comment]
Hi @dinesh2017 
In the documentation for [ngMeta](https://github.com/vinaygopinath/ngMeta/blob/master/README.md)
You have the following message:

```
TL;DR: ngMeta helps the Google crawler render your Angular site and read the meta tags. For other sites like Facebook/Twitter that can't render Javascript, you need to use pre-renderers or server redirects.
```

You can check ngMeta's documentation on how to enable this (you'll have to use some prerendering services + they have an article about that)
Cheers!
